client-core/src/main/java/com/influxdb/query/FluxRecord.java

```diff
@@ -50,6 +51,8 @@ public final class FluxRecord implements Serializable {
      */
     private LinkedHashMap<String, Object> values = new LinkedHashMap<>();
 
+    private ArrayList<Object> row = new ArrayList<>();
```

Comment:
In Java world is preferred way to use generic interface and left side of assignment:

```suggestion
    private List<Object> row = new ArrayList<>();
```